initializr-metadata/src/main/java/io/spring/initializr/metadata/InitializrConfiguration.java

```diff
@@ -155,6 +161,10 @@ private static boolean hasInvalidChar(String text) {
 		return false;
 	}
 
+	private static boolean hasReservedKeyword(final String packageName) {
+		return Arrays.stream(packageName.split("\\.")).anyMatch(SourceVersion::isKeyword);
```

Comment:
I know of the improvements made in Boot 2.2 by **not** using streams, but I doubt this is in a tight loop where it will have much impact.

Reply:
It's fine.